package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-03-fs-async",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab-03-fs-async ======",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kyleschnirring/lab-03-fs-async.git"
+  },
+  "author": "kyle schnirring",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/kyleschnirring/lab-03-fs-async/issues"
+  },
+  "homepage": "https://github.com/kyleschnirring/lab-03-fs-async#readme",
+  "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
You should add 'gulp', 'gulp-eslint', and 'gulp-mocha' to these devDependencies.